### PR TITLE
Fix the cosigned namespace to be default cosign-system

### PR DESCRIPTION
**Pull request by harshad16**
Fix the cosigned namespace to be default cosign-system
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

The `cosign-system` namespace is default and defined in operate-first apps [namepsaces](https://github.com/operate-first/apps/blob/0c4d42abdf446a0ebd60a0d2fb85f16648193492/cluster-scope/base/core/namespaces/cosigned/namespace.yaml#L6)

### CODE REVIEW

File: cluster-scope/overlays/prod/moc/smaug/kustomization.yaml [hunk 1]
```diff
@@ -24,7 +24,6 @@ resources:
 - ../../../../base/core/namespaces/aicoe-meteor
 - ../../../../base/core/namespaces/apicurio-apicurio-registry
 - ../../../../base/core/namespaces/b4mad-racing
-- ../../../../base/core/namespaces/cosigned
```
Comment (4n4nd) [inline cluster-scope/overlays/prod/moc/smaug/kustomization.yaml:27]: should be renamed instead of being removed

**Review by 4n4nd — CHANGES_REQUESTED** [review]

File: cluster-scope/overlays/prod/moc/smaug/kustomization.yaml (hunk 1 above)
Comment (harshad16) [reply cluster-scope/overlays/prod/moc/smaug/kustomization.yaml:27]: this is currently not needed in smaug cluster. 
removed for now, will add it when required.

Comment (4n4nd) [reply cluster-scope/overlays/prod/moc/smaug/kustomization.yaml:27]: oops looks like this needs to be removed from the smaug cluster.

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**